### ISSUE TITLE
Crash when playing movies

### ISSUE THREAD

**Issue by Enjoy-Combi** [nextcloud/android#7519]
### Steps to reproduce
1. Start nextcloud on android 10
2. Lunch the movie provided with the app.
3. Put it full screen

### Expected behaviour
- see the whole movie

### Actual behaviour
- Crash after few seconds after starting the movie

### Can you reproduce this problem on https://try.nextcloud.com?
- no issue

### Environment data
Android version: 10

Device model: Huawei p30 pro

Stock or customized system:
Stock

Nextcloud app version:
3.14.0

Nextcloud server version:
20.0.1
Reverse proxy:
Haproxy
### Logs
************ CAUSE OF ERROR ************

android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{4638cac u0 com.nextcloud.client/.media.PlayerService}
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2400)
	at android.os.Handler.dispatchMessage(Handler.java:110)
	at android.os.Looper.loop(Looper.java:219)
	at android.app.ActivityThread.main(ActivityThread.java:8347)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1055)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: HUAWEI
Device: HWVOG
Model: VOG-L29
Id: HUAWEIVOG-L29
Product: VOG-L29EEA

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 10.1.0.150C431

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
Fixed via 3.14.1 RC2, which is available via play store beta program.

**Comment by Enjoy-Combi** [comment]
Good, thank you

Le ven. 27 nov. 2020 à 07:02, Tobias Kaminsky <notifications@github.com> a
écrit :
